ministryofjustice/laa-apply-for-criminal-legal-aid
# CRIMAPP-891 Unable to submit because passporting benefit section appears incorrectly incomplete

## Description of change

Validate passporting benefit check based on outcome not answers.
Also shifts testing details of TypeOfMeansAssessment to the concern.
Uses the answer validator for completed and not_applicable  status in the task list, in preparation for using the same logic to decide which answers are displayed/submitted to the datastore.

## Link to relevant ticket

[CRIMAPP-891](https://dsdmoj.atlassian.net/browse/CRIMAPP-891)

## Notes for reviewer

## Screenshots of changes (if applicable)

### Before changes:

### After changes:

## How to manually test the feature

See steps in ticket

[CRIMAPP-891]: https://dsdmoj.atlassian.net/browse/CRIMAPP-891?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
